Xamarin.PropertyEditing.Mac/Controls/DateTimeEditorControl.cs

```diff
@@ -16,7 +16,8 @@ public DateTimeEditorControl (IHostResourceProvider hostResources)
 				DatePickerElements = NSDatePickerElementFlags.HourMinuteSecond | NSDatePickerElementFlags.YearMonthDateDay,
 				DatePickerStyle = NSDatePickerStyle.TextFieldAndStepper,
 				Font = NSFont.SystemFontOfSize (NSFont.SystemFontSizeForControlSize (NSControlSize.Small)),
-				TranslatesAutoresizingMaskIntoConstraints = false
+				TranslatesAutoresizingMaskIntoConstraints = false,
+				TimeZone = Foundation.NSTimeZone.FromAbbreviation ("GMT")

```

Comment:
Sorry about the late reply. Just wondering why we want to hard code this to "GMT"?  
Could you also give an example of the inconsistency you are seeing, so I have something to compare to?

Reply:
This should be UTC not GMT.


![image](https://user-images.githubusercontent.com/1587480/90165691-57250480-dd99-11ea-82ca-101f8cd760cb.png)

Reply:
This should be UTC, not GMT. Add the abbreviation into a constant.

![9qYGQXRAjZ](https://user-images.githubusercontent.com/1587480/90165958-c3a00380-dd99-11ea-9918-8969f55b9d77.gif)

Reply:
As the recent conversation this doesn't fix the real problem in Xam.Forms and it needs to be addressed in different way, adding some customisation to allow, in addition to full date time, only time or date.